.changelog/1821.txt

```diff
@@ -0,0 +1,3 @@
+```release-note:improvement
+core: git poller ignores changes outside of the configured project path
```

Comment:
```suggestion
core: git poller optionally ignores changes outside of the configured project path
```

Reply:
@mitchellh I guess you mistakenly reverted @briancain's suggestion :S